DiscordBot.DataAccess/IEventsSheetsService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using DiscordBot.Models;
+
+namespace DiscordBot.DataAccess
+{
+    interface IEventsSheetsService
```

Comment:
I would expect this to live in the same file as the implementation class 🙂

Reply:
Yep I agree, especially when there isn't a dedicated mock service. Changed in 729f3d0 🙂